src/camvidlog2/vid.py

```
@@ -161,20 +167,53 @@
         FrameError: If unable to seek to the requested position
     """
     video_capture = cv2.VideoCapture(str(filename), cv2.CAP_ANY)
+    try:
+        # cv2.CAP_PROP_POS_FRAMES is unreliable - see https://github.com/opencv/opencv/issues/9053
+        if start_ms is not None:
+            set_success = video_capture.set(cv2.CAP_PROP_POS_MSEC, start_ms)
+            if not set_success:
+                raise FrameError("Unable to seek video file")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)